lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,19 @@
 import React, { Component } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
should get rid of a console warning. No use for `{ Component }` here since Cards.js is a functional component.